src/SessionList.ts

```diff
@@ -2,111 +2,256 @@
-    $trEnter.append('td').text((d) => d.creator);
-    //$tr_enter.append('td').text((d) => `${d.size[0]} / ${d.size[1]}`);
-    $trEnter.append('td').html((d) => {
-      return `<button class="btn btn-xs btn-default" data-action="select" ${!isLoggedIn() && !d.local ? 'disabled="disabled"' : ''}><span class="fa fa-folder-open" aria-hidden="true"></span> Select</button>
-      <button class="btn btn-xs btn-default" data-action="clone"><span class="fa fa-clone" aria-hidden="true"></span> Clone</button>
-      <button class="btn btn-xs btn-default" data-action="delete" ${!isLoggedIn() && !d.local ? 'disabled="disabled"' : ''}><i class="fa fa-trash" aria-hidden="true"></i> Delete</button>`;
-    });
 
-    $trEnter.select('button[data-action="delete"]').on('click', async (d) => {
-      const deleteIt = await areyousure('Are you sure to delete: "' + d.name + '"');
-      if (deleteIt) {
-        this.targid.graphManager.delete(d);
-        $table.selectAll('#'+d.id).remove();
-      }
-    });
-    $trEnter.select('button[data-action="clone"]').on('click', (d) => {
-      this.targid.graphManager.loadOrClone(d, false);
-      return false;
-    });
-    $trEnter.select('button[data-action="select"]').on('click', (d) => {
-      this.targid.graphManager.loadOrClone(d, true);
-      return false;
+  protected async build(manager: CLUEGraphManager) {
+
+    const $parent = this.createLoader();
+
+    //select and sort by date desc
+    const workspaces = (await manager.list()).filter((d) => isPersistent(d)).sort(byDateDesc);
+
+    const me = currentUserNameOrAnonymous();
+    const myworkspaces = workspaces.filter((d) => d.creator === me);
+    const otherworkspaces = workspaces.filter((d) => d.creator !== me);
+
+    $parent.html(`
+        <ul class="nav nav-tabs" role="tablist">
+          <li class="active" role="presentation"><a href="#session_mine" class="active"><i class="fa fa-user"></i> My Sessions</a></li>
+          <li role="presentation"><a href="#session_others"><i class="fa fa-users"></i> Other Sessions</a></li>
+        </ul>
+        <div class="tab-content">
+            <div id="session_mine" class="tab-pane active">
+                <table class="table table-striped table-hover table-bordered table-condensed">
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th></th>
+                    <th>Date</th>
+                    <th></th>
+                  </tr>
+                </thead>
+                <tbody>
+            
+                </tbody>
+              </table>
+            </div>
+            <div id="session_others" class="tab-pane">
+                <table class="table table-striped table-hover table-bordered table-condensed">
+                <thead>
+                  <tr>
+                    <th>Name</th>
+                    <th>Creator</th>
+                    <th>Date</th>
+                    <th></th>
```

Comment:
I think that here an "Actions" header instead of an empty one might also be good